examples-standalone/grid-charts-integration/src/app/components/charts/common/stocks-chart.template.html

```diff
@@ -9,33 +8,35 @@ <h3> Configuration</h3>
-                    [stack]="chartConfiguration.stack" [gap]="2" [spacing]="0.25" [data]="data" [field]="series" [name]="getTitle(series)"
-                    [categoryField]="'symbol'">
-                    <kendo-chart-series-item-tooltip>
-                        <ng-template let-value="value">
-                            {{getTitle(series)}}: {{value}}
-                        </ng-template>
-                    </kendo-chart-series-item-tooltip>
-                </kendo-chart-series-item>
-            </kendo-chart-series>
+        <div class="drawer-content">
+            <kendo-chart #chart>
+                <kendo-chart-title text="Details per Stock"></kendo-chart-title>
+                <kendo-chart-series>
+                    <kendo-chart-series-item *ngFor="let series of selectedSeries" [type]="chartConfiguration.seriesType"
+                        [stack]="chartConfiguration.stack" [gap]="2" [spacing]="0.25" [data]="data" [field]="series"
+                        [name]="getTitle(series)" [categoryField]="'symbol'">
+                        <kendo-chart-series-item-tooltip>
+                            <ng-template let-value="value">
+                                {{getTitle(series)}}: {{value}}
+                            </ng-template>
+                        </kendo-chart-series-item-tooltip>
+                    </kendo-chart-series-item>
+                </kendo-chart-series>
 
-            <kendo-chart-category-axis>
-                <kendo-chart-category-axis-item [labels]="{ rotation: 'auto' }">
-                </kendo-chart-category-axis-item>
-            </kendo-chart-category-axis>
+                <kendo-chart-category-axis>
+                    <kendo-chart-category-axis-item [labels]="{ rotation: 'auto' }">
+                    </kendo-chart-category-axis-item>
+                </kendo-chart-category-axis>
 
-            <kendo-chart-value-axis>
-                <kendo-chart-value-axis-item [labels]="{ rotation: 'auto' }">
-                </kendo-chart-value-axis-item>
-            </kendo-chart-value-axis>
-            <kendo-chart-legend position="bottom" orientation="horizontal">
-            </kendo-chart-legend>
-        </kendo-chart>
-        <button kendoButton [iconClass]="'k-i-menu k-icon'" (click)="drawer.toggle()"></button>
-        <button kendoButton [iconClass]="'k-i-download k-icon'" (click)="exportChart(chart)"></button>
+                <kendo-chart-value-axis>
+                    <kendo-chart-value-axis-item [labels]="{ rotation: 'auto' }">
+                    </kendo-chart-value-axis-item>
+                </kendo-chart-value-axis>
+                <kendo-chart-legend position="bottom" orientation="horizontal">
+                </kendo-chart-legend>
+            </kendo-chart>
+            <button kendoButton [iconClass]="'k-i-menu k-icon'" (click)="drawer.toggle()"></button>
+            <button kendoButton [iconClass]="'k-i-download k-icon'" (click)="exportChart(chart)"></button>
```

Comment:
It might be better to use '_k-i-pdf_' icon here instead of '_k-i-download_'

Reply:
I have some suggestions for the icons:

- reorder them, with the drawer one on the right
- add title attribute or tooltip to the buttons - "Export as PNG" and "Configuration"
- replace the menu icon with a cog/gear one as it opens the Chart configuration.